src/components/Comment/Comment.tsx

```diff
@@ -207,116 +287,133 @@ export const Comment = ({
-              <Row>
-                {!!comment.userProfile.badge.filter(
-                  obj => obj["name"] === "Staff"
-                ).length ? (
-                  <div className={iconWrapper}>
-                    <GuardianStaff />
+        {/* Default view */}
+        <div className={css`
+          ${until.mobileLandscape} {
+            display: none;
+          }
+        `}>
+          <div className={flexRowStyles}>
+            <div className={cx(marginRight, flexGrow)}>
+              <Avatar
+                imageUrl={comment.userProfile.avatar}
+                displayName={comment.userProfile.displayName}
+                size={isReply ? "small" : "medium"}
+              />
+            </div>
+            <div className={flexColStyles}>
+              <header className={cx(flexRowStyles, headerStyles, fullWidthStyles)}>
+                <div className={flexColStyles}>
+                  <div className={cx(flexRowStyles, alignItemsCenter)}>
+                    <div className={marginRight}>
+                      <ProfilName
+                        pillar={pillar}
+                        userId={comment.userProfile.userId}
+                        displayName={comment.userProfile.displayName}
+                      />
+                    </div>
+                    <Timestamp
+                      isoDateTime={comment.isoDateTime}
+                      linkTo={joinUrl([
+                        // Remove the discussion-api path from the baseUrl
+                        baseUrl
+                          .split("/")
+                          .filter(path => path !== "discussion-api")
+                          .join("/"),
+                        "comment-permalink",
+                        comment.id.toString()
+                      ])}
+                    />
                   </div>
-                ) : (
-                  <></>
-                )}
-                {comment.status !== "blocked" && isHighlighted ? (
-                  <div className={iconWrapper}>
-                    <GuardianPick />
+                  <div className={flexRowStyles}>
+                    <Badges comment={comment} isHighlighted={isHighlighted} />
                   </div>
-                ) : (
-                  <></>
-                )}
-              </Row>
-            </Column>
-            {comment.status !== "blocked" && (
-              <RecommendationCount
-                commentId={comment.id}
-                initialCount={comment.numRecommends}
-                alreadyRecommended={false}
+                </div>
+                <>
+                  {comment.status !== "blocked" && (
+                    <RecommendationCount
+                      commentId={comment.id}
+                      initialCount={comment.numRecommends}
+                      alreadyRecommended={false}
+                    />
+                  )}
+                </>
+              </header>
+              <CommentMessage
```

Comment:
Maybe this new component could live in its own file?

Reply:
good point